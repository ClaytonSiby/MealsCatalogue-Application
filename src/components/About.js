import React from 'react'
import { Container } from 'react-bootstrap'
import projectGif from '../assets/demo/projectGif.gif'
import styles from '../assets/about.module.css';

const About = () => (
  <Container className={`${styles.aboutContainer} px-3`}>
    <h2><i>About the Application</i></h2>
    <div>
      <i>SIBYMeals</i>
      &nbsp; is an application to help users prepare meals conveniently for
      their loved ones. With our catalogue list of various foods, you can choose
      a certain category of foods &amp; view details on each listed food item.
      The detailed page for each food item gives you the cooking instructions
      &amp; a source link which gives you more information on the ingredients
      you need &amp; their prospective measurements.
    </div>
    <div className="my-3">
      <img style={{ width: '80%' }} alt='project gif' src={`${projectGif}`} />
    </div>
  </Container>
)

export default About
