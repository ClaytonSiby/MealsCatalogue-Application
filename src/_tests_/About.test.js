import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import About from '../components/About';

describe('Testing the About component', () => {
  it('Renders successfully to the DOM', () => {
    render(<About />);
  });

  test('renders output consistently to the DOM', () => {
    const test = renderer.create(<About />).toJSON();
    expect(test).toMatchSnapshot();
  });
});
