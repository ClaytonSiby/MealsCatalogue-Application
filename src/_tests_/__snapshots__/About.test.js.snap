// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the About component renders output consistently to the DOM 1`] = `
<div
  className="aboutContainer px-3 container"
>
  <h2>
    <i>
      About the Application
    </i>
  </h2>
  <div>
    <i>
      SIBYMeals
    </i>
      is an application to help users prepare meals conveniently for their loved ones. With our catalogue list of various foods, you can choose a certain category of foods & view details on each listed food item. The detailed page for each food item gives you the cooking instructions & a source link which gives you more information on the ingredients you need & their prospective measurements.
  </div>
  <div
    className="my-3"
  >
    <img
      alt="project gif"
      src="projectGif.gif"
      style={
        Object {
          "width": "80%",
        }
      }
    />
  </div>
  <p
    className="my-4 copyright"
  >
    Designed and developed by:
     
    <a
      href="https://github.com/ClaytonSiby"
      rel="noreferrer"
      target="_blank"
    >
      <i>
        Clayton Siby
      </i>
    </a>
      © copyright 2021
  </p>
</div>
`;
